modules/core/src/store/actions/types/authenticate.ts

```diff
@@ -52,7 +52,13 @@ export enum CommonAuthenticateActionTypes {
      *
      * @type {string}
      */
-    SET_DEFAULT_TENANT = "SET_DEFAULT_TENANT"
+    SET_DEFAULT_TENANT = "SET_DEFAULT_TENANT",
+    /**
+     * Redux action type to set tenants.
+     *
+     * @type {any[]}
```

Comment:
This is a string enum.
```suggestion
     * @type {string}
```